### PR TITLE
post seo tag fixes

### PR DESCRIPTION
**Pull request by altsang**

### CODE REVIEW

File: _includes/insert-to-head.html [hunk 1]
```diff
@@ -27,3 +27,10 @@
 <!-- SEO Metadata -->
 {% seo %}
 <!-- End SEO Metadata -->
+<!-- Default image for pages without page image -->
+{% if page.header-img %}
+    <meta property="og:image" content="{{ site.url }}/{{ page.header-img }}" />
```
Comment (XVincentX) [inline _includes/insert-to-head.html:32]: We already have a `expressgateway_metaimage.png`, let's replace that one instead of adding a new one.

**Review by XVincentX — CHANGES_REQUESTED** [review]
2 images out of 4 are not used and we already have got a meta image; let's replace that one instead of adding yet another one.

File: _includes/insert-to-head.html (hunk 1 above)
Comment (altsang) [reply _includes/insert-to-head.html:32]: I'll check with @shashikan8h , i think the reason for introducing a new one is that the main logo is used and that it's horizontally oriented for image inclusion in things like twitter cards, but I dunno

**Comment by altsang** [comment]
> 2 images out of 4 are not used and we already have got a meta image; let's replace that one instead of adding yet another one.

which 2 images aren't used?

File: _includes/insert-to-head.html (hunk 1 above)
Comment (altsang) [reply _includes/insert-to-head.html:32]: I updated `expressgateway_metaimage.png` with the logo file - @shashikan8h wants to make everything consistent

**Review by XVincentX — APPROVED** [review]

**Pull request merged**